user-org-util/src/main/java/org/sunbird/util/JsonKey.java

```diff
@@ -0,0 +1,914 @@
+package org.sunbird.util;
```

Comment:
change JsonKey to userOrgJsonKey
keep only those keys which are currently used.